viewer/packages/tools/src/Toolbar/ToolbarTool.ts

```diff
@@ -0,0 +1,181 @@
+    if (cameraControlOption.mouseWheelAction === 'zoomPastCursor') {
+      this._cameraOptions[1] = false;
+    } else {
+      this._cameraOptions[1] = true;
+    }
+  }
+
+  /**
+   * Add a Icon button into the Toolbar container
+   * @param backgroundImageUri Background image to be placed onto icon
+   * @param onClick Click event callback function which will be used to perform custom functionlity of the user
+   * @param toolTip Optional tooltip for the icon button
+   */
+  public addToolbarButton(backgroundImageUri: string, onClick: () => void, toolTip: string = ''): void {
+    this._toolbar.addToolbarButton(backgroundImageUri, false, onClick, false, toolTip);
+  }
+
+  /**
+   * Add a Icon toggle button into the Toolbar container
+   * @param backgroundImageUri Background image to be placed onto icon
+   * @param onToggled Click event callback function which will be used to perform custom functionlity of the user
+   * @param isActive Is the feature active by default.
+   * @param toolTip Optional tooltip for the icon button
+   */
+  public addToolbarToggleButton(
+    backgroundImageUri: string,
+    onToggled: () => void,
+    isActive: boolean,
+    toolTip: string = ''
+  ): void {
+    this._toolbar.addToolbarButton(backgroundImageUri, true, onToggled, isActive, toolTip);
+  }
+
+  /**
+   * Set the position of the toolbar container
+   * @param position ToolbarPosition value such as Top, Bottom, Left, Right within the canvas
+   */
+  public setPosition(position: ToolbarPosition): void {
+    this._toolbar.setPosition(position);
+  }
+
+  /**
+   * Add Enable or Disable Axis view tool to the Toolbar
+   */
+  public addAxisToolToggle(): void {
+    this.addToolbarToggleButton(svgAxisIcon, this._handleAxisViewToolListener, false, 'Axis Tool');
+  }
+
+  /**
+   * Add Save the screenshot of the canvas been rendered to the Toolbar
+   */
+  public addTakeScreenshotTool(): void {
+    this.addToolbarButton(svgScreenshotIcon, this._handleScreenshotListener, 'Take Screenshot');
+  }
+
+  /**
+   * Adds Toggle enabling/disabling the Camera target on mouse click to the Toolbar
+   */
+  public addCameraTargetOnClickToggle(): void {
+    this.addToolbarToggleButton(
+      svgCameraTargetIcon,
+      this._handleChangeCameraTargetListener,
+      this._cameraOptions[1],
+      'Enable/Disable Camera Target on Click'
+    );
+  }
+
+  /**
+   * Adds Toggle camera for "Zoom Past Cursor" and "Zoom To Cursor" in to Toolbar
+   */
+  public addZoomPastToCursorToggle(): void {
+    this.addToolbarToggleButton(
+      svgZoomPastIcon,
+      this._handleCameraZoomPastCursorListener,
+      this._cameraOptions[0],
+      'Toggle Zoom past/Zoom to Cursor'
+    );
+  }
+
+  /**
+   * Adds Fit Camera to Model into Toolbar
+   */
+  public addFitCameraToModel(): void {
+    this.addToolbarButton(svgFitToCameraIcon, this._handleFitCameraToModelListener, 'Fit Camera to Model');
```

Comment:
```suggestion
    this.addToolbarButton(svgFitToCameraIcon, this._handleFitCameraToModelListener, 'Fit Camera to Model');
```
Generally, don't use capital first letters. We wouldn't do this in plain english so we shouldn't in customer facing messages either.